src/Text/Pretty/Simple/Internal.hs

```diff
@@ -14,6 +14,4 @@ module Text.Pretty.Simple.Internal
 import Text.Pretty.Simple.Internal.Color as X
 import Text.Pretty.Simple.Internal.ExprParser as X
 import Text.Pretty.Simple.Internal.Expr as X
-import Text.Pretty.Simple.Internal.ExprToOutput as X
-import Text.Pretty.Simple.Internal.Output as X
```

Comment:
Two modules have gone, as we now render straight from `Expr`.

The remaining `OutputPrinter` should perhaps be renamed `ExprPrinter`, since the old `Output` type that was being printed there no longer even exists.